stylesheets/desktop/stylesheet-index.css

```diff
@@ -0,0 +1,158 @@
+
+#index-container {
+  float: left;
+  width: 40%;
+  margin-left: 15%;
+  margin-top:2%;
+  font-size: 100%;
+}
+
+#index-image {
+display: block;
+float: none;
+width: 100%;
+padding-right: 5%;
+float: left;
+margin-bottom: 5%;
+}
+
+.index-header {
+  color: #093C60;
+  text-overflow: inherit;
+  font-size: 2em;
+  margin-bottom: 3%;
+  margin-top: 5%;
+}
+
+#index-content {
+  color: #3D3D3D;
+  line-height: 2em;
+  font-size: 1em;
+}
+
+aside {
+  width: 30%;
+  float: right;
+  margin-right: 3%;
+}
+
+.aside-blog-image {
+  margin-top: 3%;
+  width: 70%;
+  height: 12rem;
```

Comment:
Try not to mix units like this, (% and rem) you can see that some of your pics look distorted. Generally you want to set the width and leave the height blank so it will auto fill and keep the aspect ratio correct.